SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -1340,35 +1350,29 @@ void USpatialActorChannel::ServerProcessOwnershipChange()
 	bool bUpdatedThisActor = false;
 
 	// Changing an Actor's owner can affect its NetConnection so we need to reevaluate this.
-	FString NewClientConnectionWorkerId = SpatialGDK::GetConnectionOwningWorkerId(Actor);
-	if (SavedConnectionOwningWorkerId != NewClientConnectionWorkerId)
+	check(NetDriver->StaticComponentView->HasAuthority(EntityId, SpatialConstants::NET_OWNING_CLIENT_WORKER_COMPONENT_ID));
+	SpatialGDK::NetOwningClientWorker* CurrentNetOwningClientData = NetDriver->StaticComponentView->GetComponentData<SpatialGDK::NetOwningClientWorker>(EntityId);
+	const FString CurrentClientWorkerId = CurrentNetOwningClientData->WorkerId.IsSet() ? CurrentNetOwningClientData->WorkerId.GetValue() : FString();
+	const FString NewClientConnectionWorkerId = SpatialGDK::GetConnectionOwningWorkerId(Actor);
```

Comment:
Why do we need worker ids anymore, can't we just use the partition id?

Reply:
because we not ditching EntityACL just yet